src/main/java/com/vaadin/demo/ce/AvatarGroupDocumentation.java

```diff
@@ -0,0 +1,65 @@
+package com.vaadin.demo.ce;
+
+import java.io.ByteArrayInputStream;
+
+import com.vaadin.collaborationengine.CollaborationAvatarGroup;
+import com.vaadin.collaborationengine.UserInfo;
+import com.vaadin.demo.domain.User;
+import com.vaadin.demo.domain.User.UserService;
+import com.vaadin.flow.component.avatar.Avatar;
+import com.vaadin.flow.component.orderedlayout.VerticalLayout;
+import com.vaadin.flow.server.StreamResource;
+
+/**
+ * Code snippets used in CollaborationAvatarGroup's reference documentation.
+ */
+public class AvatarGroupDocumentation extends VerticalLayout {
+
+    private CollaborationAvatarGroup avatarGroup;
+    private UserService userService;
+
+    public AvatarGroupDocumentation() {
+        // tag::avatar-group-new[]
+        User userEntity = userService.getCurrentUser();
+        UserInfo userInfo = new UserInfo(userEntity.getId());
+
+        CollaborationAvatarGroup avatarGroup = new CollaborationAvatarGroup(
+                userInfo, "avatars");
```

Comment:
This was changed by Peppe in the docs repo earlier, but got now overridden by bringing in these sources from the CE repo:
```suggestion
                userInfo, "ensuranceClaims");
```